### ISSUE TITLE
Creator Token Metadata

### ISSUE THREAD

**Issue by dobertRowneySr** [Joystream/joystream#4713]
- [x] add a `creator_token_remark` extrinsic for metadata support
- [x] compile list of metadata message format (see down below)
- [x] ~publish~ patch protobuf package with new metadata for CRT?
- [x] process metadata in orion v2?
- [x] Review and open PR in orion repo https://github.com/Joystream/orion/pull/99

**Comment by dobertRowneySr** [comment]
# CRT Metadata message format
``` go
syntax = "proto2";

message TokenMetadata {
  optional string name = 1; // Title
  optional string description = 2; // token description
  oneof avatar {
    uint32 avatar_object = 3; // avatar for token - index into external [assets array](#.Assets)
    string avatar_uri = 4; // Url to member's avatar
  }
  repeated Benefit benefits = 5; // benefits for tokne
  optional string whitelist_application_note = 6; // note for applicant
  optional string whitelist_application_apply_link = 7; // link to application process
}

message Benefit {
  optional string title = 1;
  optional string description = 2;
  optional string emoji = 3;
  uint32 display_order = 4;
}

message SaleMetadata {
  optional string terms_and_conditions = 1
}

```

**Comment by dobertRowneySr** [comment]
symbol should be added.
Addressed in #4871
